bark/models/behavior/action_store/behavior_action_store.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2019 fortiss GmbH, Julian Bernhard, Klemens Esterle, Patrick
+// Hart, Tobias Kessler
+//
+// This work is licensed under the terms of the MIT license.
+// For a copy, see <https://opensource.org/licenses/MIT>.
+
+#include "bark/models/behavior/action_store/behavior_action_store.hpp"
+
+
+namespace bark {
+namespace models {
+namespace behavior {
+
+using dynamic::Trajectory;
+
+
+ActionHash BehaviorActionStore::Store(const Action& action, const Trajectory& trajectory,
+                                      const BehaviorStatus& status) {
+  const auto action_hash = ActionToHash(action);
```

Comment:
why do we not just use a boost variant and use a hash here?
Without further documentation I do not understand why an action is stored like this